include/circt/Dialect/FIRRTL/FIRRTLExpressions.td

```diff
@@ -567,12 +581,13 @@ def AsPassivePrimOp : FIRRTLOp<"asPassive", [NoSideEffect]> {
 }
 
 // Convert a passive value to a non-passive type.
-def AsNonPassivePrimOp : FIRRTLOp<"asNonPassive", [NoSideEffect]> {
+def AsNonPassivePrimOp : FIRRTLOp<"asNonPassive", [NoSideEffect,
+                               PassiveConstraint<"result", "input">]> {
```

Comment:
super nit, but plz add 3 more spaces to the new line to line it up nicer.  we need tblgen-format :-)